pkg/capabilities/mercury/mercury.go

```diff
@@ -51,6 +52,21 @@ type ReportInfo struct {
 	Price     float64
 }
 
+// TODO: fix this by adding support for uint64 in value.go
+type FeedReport struct {
```

Comment:
Feel free to remove the old structures above and update the aggregator (we can also do it in a separate PR to not grow this one too much).

Reply:
Will do in a separate PR